__tests__/runner.js

```diff
@@ -1,11 +1,14 @@
 const assert = require('assert');
-const util = require('util');
 const path = require('path');
 const fs = require('fs').promises;
 const {readdirSync, readFileSync} = require('fs');
 const execa = require('execa');
 const dircompare = require('dir-compare');
 
+function stripBenchmark(stdout) {
+  return stdout.replace(/\s*\[\d+s\]$/, ''); //remove benchmark
+}
```

Comment:
We needed some way to remove the benchmarks, because fractions of seconds will throw the test. Is this OK?

Reply:
I'd made a change so that the benchmark was never output when `NODE_ENV === 'test'`, but I like this method much better than my custom test-only behavior. Nice thinking. Can you replace my NODE_ENV workaround so that all tests match?

https://github.com/pikapkg/web/blob/4cba2484e84b62fb9ada17d3da5f684a1f87c076/src/index.ts#L419